protocol/src/test/java/org/fido/iot/protocol/RendezvousInfoDecoderTest.java

```diff
@@ -0,0 +1,38 @@
+package org.fido.iot.protocol;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class RendezvousInfoDecoderTest {
+
+  @Test
+  void getHttpDirectivesTest() throws Exception {
+
+    String ownerdirective  = "http://localhost:8040?ipaddress=10.66.244.99&ownerport=8443";
```

Comment:
Please remove the IP and probably, you can add a comment that the localhost dnsname or localhost IP address can be replaced by any other IP

Reply:
Updated. Added the comment also.